helm/reana/templates/priority-classes.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if index .Values "fluent-bit" "enabled" }}
+---
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: {{ include "reana.prefix" . }}-fluent-bit-priority-class
+value: 1000000
```

Comment:
Can we keep the value 1000000 hard-coded, or do we need to make it more easily configurable by admins at the deployment time?

Reply:
Hm, maybe still worth making it configurable and add 1000000 as a default, though I doubt anyone would ever need to change this.